lib/fastlane/plugin/wpmreleasetoolkit/actions/common/get_prs_list_action.rb

```diff
@@ -60,20 +45,15 @@ def self.available_options
                                        description: 'The remote path of the GH repository on which we work',
```

Comment:
More like the repo slug (`<org name>/<reponame>`) rather than the "remote path" (What was that even supposed to mean? URL?)

Especially since this is used in the milestone filter/query and the "path"/URL would not work for that

Reply:
Yeah, good point. This was what it was originally – I just copied the description from the helper YARDdoc in 8c3e4affab2acde6067ba4af4de8d044591a350d, which should be quite clear I think